src/ocaml/mode.mli

```diff
@@ -26,10 +24,6 @@ val of_cm_kind : Cm_kind.t -> t
 
 val variant : t -> Variant.t
 
-val to_string : t -> string
-
-val to_dyn : t -> Dyn.t
```

Comment:
How come you removed these functions? They seem pretty useful to have.

Reply:
I am moving all the functions that involve serialization, encoding,... anything related to strings, to the `Lib_mode` module. 

The reason is that I thought it would make harder to run into bugs, imagine if Ocaml and Melange ended up using the same string for something.

Reply:
Another reason is backwards compatibility. If this was starting from scratch, the config would be `(modes (ocaml byte)`.

Reply:
`to_dyn` is mostly used for debugging though.

And I think that your lib mode can rely on the original `Mode.{to_dyn,to_string}` functions in their implementations.